content/010_introduction/12_problem_to_solve.md

```diff
@@ -5,6 +5,6 @@ draft: false
 weight: 12
 ---
 # What Is The Challenge/Problem?
-Traditional software releases are risky and stressful. When developers deploy code to production, that code gets released to the whole user base at the same time. In this scenario, incidents that occur affect a large audience. What's more, resolving such incidents often involves emergency fixes, rollbacks, and redeploys, some of which can be expensive and time-intensive. These risks hurt developer productivity, diminish team morale, and slow the pace of innovation.
-
+In traditional software releases, functionality is released at the time of the software is deployed. We deploy our code to an environment, and when users access it, the entire application is available for them to consume. This approach carries a lot of risk. Users consume all of the features of the application you've released, bugs and all. If larger problem arise in the deployment, its either emergency triage or a rollback (the redeployment of a previous version). Furthermore, the problem is impacting potentially your entire user community. We refer to this as the "blast radius".  
```

Comment:
"If larger problem arise in the deployment" -> deployment is the process, version or release